packages/clerk-js/src/utils/captcha/turnstile.ts

```diff
@@ -69,21 +71,38 @@ export const shouldRetryTurnstileErrorCode = (errorCode: string) => {
   return !!codesWithRetries.find(w => errorCode.startsWith(w));
 };
 
-async function loadCaptcha(url: string) {
+async function loadCaptcha(fallbackUrl: string) {
   if (!window.turnstile) {
     try {
-      await loadScript(url, { defer: true });
+      await loadCaptchaFromCloudflareURL();
     } catch {
-      // Rethrow with specific message
-      console.error('Clerk: Failed to load the CAPTCHA script from the URL: ', url);
-      throw {
-        captchaError: 'captcha_script_failed_to_load',
-      };
+      await loadCaptchaFromFAPIProxiedURL(fallbackUrl);
     }
   }
   return window.turnstile;
 }
 
+async function loadCaptchaFromCloudflareURL() {
+  try {
+    await loadScript(CLOUDFLARE_TURNSTILE_ORIGINAL_URL, { defer: true });
+  } catch (err) {
+    console.error('Clerk: Failed to load the CAPTCHA script from the original Cloudflare URL.');
+    throw err;
+  }
+}
+
+async function loadCaptchaFromFAPIProxiedURL(fallbackUrl: string) {
+  try {
+    await loadScript(fallbackUrl, { defer: true });
+  } catch {
+    // Rethrow with specific message
+    console.error('Clerk: Failed to load the CAPTCHA script from the URL: ', fallbackUrl);
+    throw {
+      captchaError: 'captcha_script_failed_to_load',
```

Comment:
This looks like it needs to be the last error thrown, meaning that if we ever change the order loadCaptchaFromFAPIProxiedURL is called in the future this will most likely break.

How about moving the throw in loadCaptcha instead?